src/translations/en.js

```diff
@@ -17,6 +17,8 @@ module.exports = {
   fieldInvalidForPostalCode: 'This postal code is not valid.',
   genericError: 'Something went wrong on our end.',
   hostedFieldsFailedTokenizationError: 'Please check your information and try again.',
+  hostedFieldsTokenizationFailOnDuplicateError: 'This credit card is already saved to a customer.',
```

Comment:
I can't think of a better option, but it might be worth checking with the designers?